### ISSUE TITLE
semantic-ui meteor not working.

### ISSUE THREAD

**Issue by Linkyz** [Semantic-Org/Semantic-UI-Meteor#134]
Hi everyone,
I did follow the step on the installation guide, and I have no css displaying.
Do I have to import any css code in my app?
And btw, I also have the warning 
`"app/client/semantic-ui/semantic.less.css: warn: There are some @import rules those are not taking effect as they are required to be in the beginning of the file." when meteor is refreshing.`
What can I do?
Did I miss something?

**Comment by myktra** [comment]
Nope, I believe as long as you ensure `custom.semantic.json` is in a `/client` folder relative to your `meteor` folder you should be good. My setup is done exactly as recommended (with `custom.semantic.json` located in `/client/lib/semantic-ui`).

Warning is already documented in #132.

**Comment by digigm** [comment]
I have the same problem with no css styling displayed. (I fix the warn above). 
I do have `custom.semantic.json` located in `/client/lib/semantic-ui`
and it looks like this:
{
  "definitions": {
    "site": true
  },
  "themes": {
    "default": true,
    "basic": true
  }
}
The files were regenerate but still can't see styling.
This is how the web page look like:

<img width="536" alt="screenshot 2018-08-15 09 00 59" src="https://user-images.githubusercontent.com/8149778/44135545-c1b9022a-a069-11e8-9e04-7caf6e3d3f73.png">

Any ideas where to look/check?

**Comment by digigm** [comment]
I had similar problem until I realised that I need to add specific component definitions to `custome.semantic.json` like this for button if I want to see buttons styled:
```
"definitions": {
    "site": true,
    "button": true
  },
```

I would be great adding that explicit step to instructions. I could not find this anywhere.  The only thing I found was:

> Edit the file custom.semantic.json to select **only the definitions and themes you want**
> Example: set themes basic to true, leave default as true

And example is missing the point of adding component like button e.g.
